scalableminds/webknossos
# Add delete segment data update action

### URL of deployed dev instance (used for testing):
- https://deletesegment.webknossos.xyz

### Steps to test:
- Delete segments in WK. You can ensure that they are deleted visually and via fossildb.

### TODOs:
- [x] Frontend: Add remove action to context menu

### Issues:
- fixes #7408 

------
(Please delete unneeded items, merge only when none are left open)
